3.20.0/rockcraft.yaml

```diff
@@ -25,10 +25,13 @@ parts:
     override-build: |
       cd chaoscenter/authentication
       CGO_ENABLED=0 go build -o "${CRAFT_PART_INSTALL}/bin/server" -v ./api/
+      # write the workload version into a VERSION file
+      echo "$(craftctl get version)" > "${CRAFT_PART_INSTALL}/VERSION"
```

Comment:
Not specifically related to this PR, but should we also reference the `version` in the `source-tag` to make sure we don't get out of sync?